lightspeed/modules/con_telemetry-data-collection-notice.adoc

```diff
@@ -50,4 +50,4 @@ Red Hat may share telemetry data with its business partners in an aggregated for
 Red Hat may engage certain service providers to assist in the collection and storage of the telemetry data.
 
 === User Control/ Enabling and Disabling Admin Dashboard Telemetry Collection
-You cannot disable collection of operational telemetry data. Operational telemetry data includes only data that is necessary to operate and troubleshoot the service. However, you can disable the collection of Admin Dashboard telemetry data. For more information, see xref:disable-admin-dashboard-telemetry_managing-admin-dashboard-telemetry[Disabling the Admin dashboard telemetry]. 
\ No newline at end of file
+You cannot disable collection of operational telemetry data. Operational telemetry data includes only data that is necessary to operate and troubleshoot the service. However, you can disable the collection of Admin Dashboard telemetry data. For more information, see xref:disable-admin-dashboard-telemetry_administering-ansible-lightspeed[Disabling the Admin dashboard telemetry]. 
```

Comment:
Just an ignorable suggestion: - not very important.
The nesting level of these sections is quite deep - 1.4.4.2.5. Maybe consider non-indexed headings for these short sections, or putting them in a Definition list.

Reply:
I edited the section titles to discrete headings (reference: https://docs.asciidoctor.org/asciidoc/latest/blocks/discrete-headings/). It looks okay in asciidoctor output but a tad bit smaller in ccutil output. If this edit is okay with you, let's see how it renders in the output and then update if required?